apps/web/src/server/lib/helpers.js

```diff
@@ -46,3 +47,24 @@ export const appealSiteObjectToText = (appealSite, separator) => {
 
 	return addressString;
 };
+
+/**
+ * Create an object of html strings containing file download links for consumption in nunjucks template, using the supplied documents data array from the LPA service
+ *
+ * @param {Array<object>} documents - array of document data items from API endpoint
+ * @returns {Object} - object containing html strings for consumption in nunjucks template
+ */
+export const makeDownloadLinksDataForTemplate = (documents) => {
```

Comment:
Worth noting, I was debating putting this logic in the templates. I'm open to suggestions if anyone thinks it would be worth moving it there (probably as a custom filter)

Reply:
I would say it is better as a template specific thing.

Reply:
This has been removed now in favour of makeDocumentLinksForType macro